### PR TITLE
Node Properties

### PR DESCRIPTION
**Pull request by georgemillo**
[Trello card](https://trello.com/c/09iYSskk/35-2-make-acunetix-use-the-node-properties)

Questions:
1. Have I got all the right properties?
2. Is my `Scan#service` method in the right format?

I'd also argue that the method `Importer#process_scan` should be extracted to a method called `Scan#process`, but that can be saved for another day.

### CODE REVIEW

**Comment by rilla** [comment]
Properties aren't added to the node in my tests. This is how I'm testing:

1. Create an empty project
2. Import one of the files included in `specs/fixtures` with: `PROJECT_ID=1 thor dradis:plugins:acunetix:upload ../community/dradis-acunetix/spec/fixtures/files/simple.acunetix.xml`
3. Open the node in the UI
4. Properties are empty

Can you check it, @georgemillo?

File: lib/acunetix/scan.rb [hunk 1]
```diff
@@ -45,30 +45,57 @@ def method_missing(method, *args)
...
+
+    def start_url_port
+      start_uri.port
+    end
+
+
+    def to_node(content_service)
+      node = content_service.create_node(label: start_url_host, type: :host)
+      node.set_property(:hostname, hostname)
+      node.set_property(:service,  service)
```
Comment (rilla) [inline lib/acunetix/scan.rb]: The property should be named `services`, and the content of the service should be splitted into meaningful parts (such as port, protocol, product, version, etc) so the node properties engine can build a table and keep track of the different services properly. Take a look at the implementation in dradis-nmap: https://github.com/dradis/dradis-nmap/blob/master/lib/dradis/plugins/nmap/importer.rb#L47

**Comment by georgemillo** [comment]
whoops

**Comment by rilla** [comment]
+1

**Pull request closed without merging**